src/Tickets/Commerce/Order_Modifiers/Table_Views/Order_Modifier_Table.php

```diff
@@ -164,7 +164,7 @@ protected function column_default( $item, $column_name ) {
 		$method = 'render_' . $column_name . '_column';
 
 		// If a specific method exists for the column, call it.
-		if ( method_exists( $this, $method ) ) {
+		if ( method_exists( $this, $method ) && is_callable( [ $this, $method ] ) ) {
```

Comment:
My comment on the other PR may have been missed, but I think we could do with *only* `is_callable()` here. That way, if `__call()` is implemented instead of an actual method definition, this will return the correct result.

```suggestion
		if ( is_callable( [ $this, $method ] ) ) {
```

Reply:
This actually breaks the tests - so ill keep `method_exists` as well.